### ISSUE TITLE
account_banking_pain_base issue with Unidecode 1.3.5

### ISSUE THREAD

**Issue by remi-filament** [OCA/bank-payment#963]
When updating today, I had an issue I could not understand, which seems related to Unidecode v1.3.5 published on Sep, 28th 2022.

## Module
account_banking_pain_base

## Describe the bug
When starting or updating Odoo I got the following error:
```
odootest | Traceback (most recent call last):
odootest |   File "/opt/odoo/odoo/odoo/http.py", line 1470, in dispatch
odootest |     odoo.registry(db).check_signaling()
odootest |   File "/opt/odoo/odoo/odoo/__init__.py", line 117, in registry
odootest |     return modules.registry.Registry(database_name)
odootest |   File "/opt/odoo/odoo/odoo/modules/registry.py", line 62, in __new__
odootest |     return cls.new(db_name)
odootest |   File "/opt/odoo/odoo/odoo/modules/registry.py", line 86, in new
odootest |     odoo.modules.load_modules(registry._db, force_demo, status, update_module)
odootest |   File "/opt/odoo/odoo/odoo/modules/loading.py", line 417, in load_modules
odootest |     force, status, report, loaded_modules, update_module, models_to_check)
odootest |   File "/opt/odoo/odoo/odoo/modules/loading.py", line 313, in load_marked_modules
odootest |     perform_checks=perform_checks, models_to_check=models_to_check
odootest |   File "/opt/odoo/odoo/odoo/modules/loading.py", line 179, in load_module_graph
odootest |     load_openerp_module(package.name)
odootest |   File "/opt/odoo/odoo/odoo/modules/module.py", line 368, in load_openerp_module
odootest |     __import__('odoo.addons.' + module_name)
odootest |   File "<frozen importlib._bootstrap>", line 969, in _find_and_load
odootest |   File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
odootest |   File "<frozen importlib._bootstrap>", line 664, in _load_unlocked
odootest |   File "<frozen importlib._bootstrap>", line 634, in _load_backward_compatible
odootest |   File "/opt/odoo/odoo/odoo/modules/module.py", line 82, in load_module
odootest |     exec(open(modfile, 'rb').read(), new_mod.__dict__)
odootest |   File "<string>", line 5, in <module>
odootest |   File "/opt/odoo/additional_addons/account_banking_pain_base/models/__init__.py", line 2, in <module>
odootest |     from . import account_payment_order
odootest |   File "/opt/odoo/additional_addons/account_banking_pain_base/models/account_payment_order.py", line 15, in <module>
odootest |     from unidecode import unidecode
odootest |   File "/usr/local/lib/python3.5/dist-packages/unidecode/__init__.py", line 151
odootest |     __import__(f'unidecode.{file.stem}', globals(), locals(), ['data']).data,
odootest |                                       ^
odootest | SyntaxError: invalid syntax
```
I had no time to investigate further, I forced previous version 1.3.4 and it got back to work !

## To Reproduce
**Affected versions**: tested on v12, may impact other as well

1. Update unidecode to latest available version
2. Restart Odoo

**Expected behavior**
Odoo starting normally

**Comment by remi-filament** [comment]
Issue opened on Unidecode official mirror on github as well : https://github.com/avian2/unidecode/issues/82
It seems the issue is not present on Odoo 14 with Python 3.9, so it may be related to Python 3.5 although the Unidecode states compatibility with python >= 3.5

**Comment by remi-filament** [comment]
Issue fixed in Unidecode with version 1.3.6, version 1.3.5 should not be used since wrongly built !
